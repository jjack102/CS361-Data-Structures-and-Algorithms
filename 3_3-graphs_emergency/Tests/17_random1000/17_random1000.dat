0 999 4
0 323 13 0
323 245 10 0
245 803 4 0
803 900 8 1
900 572 7 0
572 690 3 0
690 108 11 0
108 16 10 0
16 632 10 0
632 49 4 0
49 287 1 0
287 125 7 0
125 761 3 0
761 552 18 0
552 640 8 0
640 164 1 0
164 903 12 0
903 972 2 0
972 254 12 1
254 443 3 0
443 379 4 1
379 894 16 0
894 525 9 0
525 739 3 0
739 815 16 0
815 127 8 0
127 869 13 0
869 409 11 0
409 559 10 1
559 453 5 1
453 229 6 1
229 601 20 1
601 698 15 1
698 33 11 0
33 502 13 0
502 272 15 0
272 723 10 0
723 610 1 0
610 288 19 0
288 75 19 0
75 378 20 0
378 294 1 0
294 201 4 0
201 140 18 1
140 565 4 0
565 560 5 1
560 304 7 0
304 470 17 1
470 252 8 0
252 999 10 0
0 277 12 0
277 913 7 0
913 631 18 0
631 173 15 0
173 158 1 0
158 90 2 0
90 707 11 0
707 285 4 0
285 678 17 1
678 835 2 0
835 563 15 0
563 132 20 0
132 65 1 0
65 883 6 0
883 550 15 0
550 816 19 0
816 386 11 1
386 541 3 0
541 259 15 0
259 715 8 0
715 829 3 1
829 54 7 1
54 94 10 0
94 124 11 0
124 255 14 0
255 952 18 1
952 689 10 0
689 534 2 0
534 976 5 0
976 878 18 0
878 786 14 0
786 254 1 1
254 512 13 0
512 137 16 0
137 146 12 0
146 670 14 0
670 945 3 0
945 854 11 1
854 675 5 1
675 625 3 0
625 409 4 0
409 957 18 1
957 476 9 0
476 475 15 0
475 842 10 0
842 27 8 0
27 11 11 0
11 947 11 0
947 568 3 0
568 999 2 0
