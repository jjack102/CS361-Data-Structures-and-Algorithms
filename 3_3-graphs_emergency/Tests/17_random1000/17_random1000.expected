Fastest time for car at 0 to the emergency at 999 is 314 minutes.
